protocols/mdns/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 0.43.1
 
 - Derive `Clone` for `mdns::Event`. See [PR 3606].
+- Remove deprecated `Mdns` prefixed items. See [PR 3699].
```

Comment:
Should go under a `0.44.0` heading.

Reply:
Done, I need get back into the mode of working on this repo...